src/buildApplication/buildApplication.ts

```diff
@@ -32,10 +32,17 @@ export async function buildApplication({
 	if (!skipBuild) {
 		try {
 			await buildVercelOutput();
-		} catch (err) {
+		} catch {
 			cliError(
-				(err as Error)?.message ??
-					'Error: The Vercel build failed. For more details see the Vercel logs above.'
+				`
+					The Vercel build command failed. For more details see the Vercel logs above.
+					
+					Note: The Vercel building process is run untouched by @cloudflare/next-on-pages,
```

Comment:
I like the intent here but this feels a little verbose. I think we should drop the first sentence and just say `If you need help solving the issue, refer to the Vercel or Next.js documentation`

Reply:
I totally agree that it is verbose.... but I'm kind of trying to make it crystal clear here that the issue is not at all related to next-on-pages (so that we don't get extra issues etc...)

I'm afraid that removing the first sentence could make the message less clear? 😢